util/registry_utils.test.ts

```diff
@@ -214,27 +215,14 @@ test("isReadme", () => {
     ["READTHIS.org", false],
   ];
   for (const [path, expectedToBeReadme] of tests) {
-    expect([path, isReadme(path)]).toEqual([path, expectedToBeReadme]);
+    assertEquals([path, isReadme(path)], [path, expectedToBeReadme]);
   }
 });
 
-test("parseNameVersion", () => {
-  expect(parseNameVersion("ms@v0.1.0")).toEqual(["ms", "v0.1.0"]);
-  expect(parseNameVersion("xstate@xstate@4.25.0")).toEqual([
+Deno.test("parseNameVersion", () => {
+  assertEquals(parseNameVersion("ms@v0.1.0"), ["ms", "v0.1.0"]);
+  assertEquals(parseNameVersion("xstate@xstate@4.25.0"), [
     "xstate",
     "xstate@4.25.0",
   ]);
 });
-
-test("parseQuery", () => {
-  expect(parseQuery(["std@0.101.0", "http", "server.ts"])).toEqual({
-    name: "std",
-    version: "0.101.0",
-    path: "/http/server.ts",
-  });
-  expect(parseQuery(["oak@v9.0.1", "mod.ts"])).toEqual({
-    name: "oak",
-    version: "v9.0.1",
-    path: "/mod.ts",
-  });
-});
```

Comment:
This test is getting removed?

Reply:
parseQuery doesnt exist anymore, as we use urlpattern now. want me to write a new test?